server/athenian/api/internal/jira.py

```diff
@@ -162,6 +173,60 @@ async def get_jira_installation_or_none(
         return None
 
 
+class JIRAEntitiesMapper:
+    """Maps various JIRA issue entities from their labels to their identifiers.
+
+    Issue priorities and issue types are mapped by this object.  Project are instead mapped
+    through JIRAConfig.
+    """
+
+    def __init__(
+        self,
+        priority_names_map: dict[str, list[str]],
+        types_map: dict[str, list[str]],
+    ) -> None:
+        """Build a mapper using the given dictionaries as mapping source."""
+        self._priority_names_map = priority_names_map
+        self._types_map = types_map
+
+    @classmethod
+    async def build(cls, jira_acc_id: int, mdb: DatabaseLike) -> JIRAEntitiesMapper:
```

Comment:
```suggestion
    async def load(cls, jira_acc_id: int, mdb: DatabaseLike) -> JIRAEntitiesMapper:
```
or `fetch()` - those two are used in the rest of the code.